textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexInfo.java

```diff
@@ -78,5 +78,98 @@ static RegexInfo anyChar() {
 		regexInfo.emptyable = false;
 		return regexInfo;
 	}
+	
+	/**
+	 * This function simplifies the regexInfo. <br>
+	 * If there the "exact" set gets too large, it will add exact to the query tree,
```

Comment:
`exact` is one of the five variables (emptyable, prefix, suffix, exact, match) we compute for a regex. It is a member of RegexInfo.